web3/eth/async_eth.py

```diff
@@ -232,7 +232,7 @@ async def fee_history(
         self,
         block_count: int,
         newest_block: Union[BlockParams, BlockNumber],
-        reward_percentiles: Optional[List[float]] = None,
+        reward_percentiles: Optional[List[float]] = [],
```

Comment:
you may want to be careful defaulting to a mutable variable in a function signature!

https://softwareengineering.stackexchange.com/questions/157373/python-mutable-default-argument-why